package.json

```diff
@@ -61,10 +62,15 @@
     ]
   },
   "config": {
-    "commig-msg": "validate-commit-msg",
-    "pre-commit": "npm run validate",
+    "opt": {
```

Comment:
Please remove this from the config. This essentially doesn't allow us to have our own preferences.